installinator-common/src/progress.rs

```diff
@@ -66,6 +66,8 @@ pub enum InstallinatorComponent {
 pub enum InstallinatorStepId {
     Download,
     Format,
+    Scan,
+    UnpackControlPlaneArtifact,
```

Comment:
As discussed in our chat, it would be really nice to have a comment here explaining why we unpack control plane zones here rather than in wicketd (when we unpack other artifacts in wicketd).

Reply:
Added in 18175672